android/app/src/main/cpp/app/organicmaps/util/Language.cpp

```diff
@@ -9,4 +9,10 @@ Java_app_organicmaps_util_Language_nativeNormalize(JNIEnv *env, jclass type, jst
   std::string locale = languages::Normalize(jni::ToNativeString(env, lang));
   return jni::ToJavaString(env, locale);
 }
+
+JNIEXPORT void JNICALL
+Java_app_organicmaps_util_Language_nativeRefreshSystemLocale(JNIEnv *env, jclass type)
```

Comment:
nit: do not specify unused variable names.
```suggestion
Java_app_organicmaps_util_Language_nativeRefreshSystemLocale(JNIEnv *, jclass)
```